lib/arclight/custom_document.rb

```diff
@@ -10,12 +10,15 @@ class CustomDocument < SolrEad::Document
       t.unitid(path: 'archdesc/did/unitid', index_as: %i[displayable])
       t.repository(path: 'archdesc/did/repository/corpname/text() | archdesc/did/repository/name/text()', index_as: %i[displayable facetable])
       t.creator(path: "archdesc/did/origination[@label='creator']/*/text()", index_as: %i[displayable facetable])
+      t.prefercite(path: 'archdesc/prefercite/p', index_as: %i[displayable])
 
       # overrides of solr_ead to get different `index_as` properties
       t.extent(path: 'archdesc/did/physdesc/extent', index_as: %i[displayable])
       t.unitdate(path: 'archdesc/did/unitdate', index_as: %i[displayable])
       t.accessrestrict(path: 'archdesc/accessrestrict/p', index_as: %i[displayable])
       t.scopecontent(path: 'archdesc/scopecontent/p', index_as: %i[displayable])
+      t.userestrict(path: 'archdesc/userestrict/p', index_as: %i[displayable searchable])
+      t.abstract(path: 'archdesc/did/abstract', index_as: %i[displayable searchable])
```

Comment:
I see that `solr_ead` already parses these as `searchable` -- doesn't `displayable` also include the searchable functionality? i.e., we just need displayable here? I don't understand re: Solrizer/Solr to distinguish them.